site2/docs/install-deploy-upgrade-landing.md

```diff
@@ -0,0 +1,18 @@
+---
+Id: install-deploy-upgrade-landing
+title: Install-Deploy-Upgrade

```

Comment:
```suggestion
title: Install, deploy, and upgrade Pulsar
```
1. Do we need to make it more natural?
2. Do we need to split it into independent chapters? (that is, one chapter for `install`, one for `deploy`, and one for `upgrade`?)

Reply:
No, this needs to match the button text and only needs to be a jumping-off point for the topics listed.

Reply:
I've looked at this again and I think you are correct about the title. It should be more natural. But there is no need to split it into independent chapters because it is only a jumping-off point for other topics.